main.go

```diff
@@ -32,16 +34,15 @@ func main() {
 	if err != nil {
 		errAndExit("Improperly formatted JSON file: %v", err)
 	}
-	fmt.Println("Please input your master password")
-	var reader = bufio.NewReader(os.Stdin)
-	password, err := reader.ReadBytes('\n')
+	fmt.Print("Please input your master password: ")
+	password, err := term.ReadPassword(int(syscall.Stdin))
```

Comment:
Explicit conversion to `int` is redundant here, I believe

Reply:
Good catch.